platform/ui/src/components/ViewportOverlay/ViewportOverlay.css

```diff
@@ -10,6 +10,12 @@
   text-shadow: 1px 1px #000;
   pointer-events: none;
 }
+.overlay-top {
+  top: 28px;
+}
+.overlay-bottom {
+  bottom: 28px;
+}
 .ViewportOverlay .top-left {
```

Comment:
I believe many of the css rules (like these below) were NOT being used even before the changes for this PR. Should they be removed?

Reply:
sure